### ISSUE TITLE
Cleanup: delete unused files

### ISSUE THREAD

**Issue by jameslaneconkling** [crowdcover/congo-mines-app#92]
Low priority, but should be done before we ship the site out.  All files that aren't necessary/don't need to be kept around should be deleted.  Want everything to look clean-ish in case other developers start poking around the code.

**Comment by jameslaneconkling** [comment]
@markmcdonald51 can we remove `views/shared/documents.html.slim`, `views/shared/documents2.html.slim` and `views/searches/show.html.slim`?
